backend/src/apis/gms/model/GmsUser.ts

```diff
@@ -49,14 +49,16 @@ export class GmsUser {
     if (
       user.gmsAllowed &&
       user.alias &&
-      user.gmsPublishName === PublishNameType.PUBLISH_NAME_ALIAS_OR_INITALS
+      (user.gmsPublishName as PublishNameType) === PublishNameType.PUBLISH_NAME_ALIAS_OR_INITALS
```

Comment:
What has been changed to add this explicit "as PublishNameType"?
It seems to be necessary on every usage of enums in the following files...
It wasn't necessary before...